app/assets/javascripts/common/patient_breakdown_reports.js

```diff
@@ -109,7 +109,7 @@ PatientBreakdownReports = function () {
         totalPatientsNode.innerHTML = reports.formatNumberWithCommas(totalPatients);
         periodStartNode.innerHTML = period.ltfu_since_date;
         registrationsNode.innerHTML = reports.formatNumberWithCommas(cumulativeAssignedPatients);
-        registrationsPeriodEndNode.innerHTML = period.start_date;
+        registrationsPeriodEndNode.innerHTML = period.bp_control_end_date;
```

Comment:
I wanna learn more about the JS implementation - where does this data go and sit? Is it globbed onto data attributes, and it powers the mouseover behavior of the charts?

Reply:
[This ChartJS config](https://github.com/simpledotorg/simple-server/blob/2baa53212c1a027f352856793cb4d7f373506c42/app/assets/javascripts/common/patient_breakdown_reports.js#L87) allows us to pass it a function that gets called on the mouseover. The `innerHTML` in this function changes the numbers at the top of the card.
<img width="548" alt="Screenshot 2021-03-11 at 1 36 53 PM" src="https://user-images.githubusercontent.com/16774200/110755484-0615ba00-826f-11eb-97ab-f90d16d231a7.png">